docs/api/plugins.md

```diff
@@ -525,6 +531,46 @@ requests. It defaults to `503 ServiceUnavailableError`.
 This plugin should be registered as early as possibly in the middleware stack
 using `pre` to avoid performing unnecessary work.
 
+## CPU Usage Throttling
+
+```js
+var restify = require('restify');
+var errors = require('restify-errors');
+
+var server = restify.createServer();
+const options = {
+  limit: .75,
+  max: 1,
+  interval: 250,
+  halfLife: 500,
+  err: new errors.InternalServerError()
+}
+
+server.pre(restify.plugins.cpuUsageThrottle(options));
+```
+
+cpuUsageThrottle is a middleware that rejects a variable number of requests (between 0% and 100%) based on a historical view of CPU utilization of a Node.js process. Essentially, this plugin allows you to define what constitutes a saturated Node.js process via CPU utilization and it will handle dropping a % of requests based on that definiton. This is useful when you would like to keep CPU bound tasks from piling up causing an increased per-request latency.
```

Comment:
For bonus points might want to link to the ewma paper, but if you're already doing that in the ewma module, then maybe not. I leave that up to you :)

Reply:
Done in the EWMA module :smile: the :rabbit: :hole: is deep :wink: